docs/docs/architecture/attestation.md

```diff
@@ -341,32 +341,50 @@ When an initialized node tries to join another cluster, its measurements inevita
 
-## Chain of trust
+## Putting it all together
+In this section, we put the aforementioned concepts together and illustrate how trust into a Constellation cluster is established and maintained.
 
-So far, this page described how an entire Constellation cluster can be verified using hardware attestation capabilities and runtime measurements.
-The last missing link is how the ground truth in the form of runtime measurements can be securely distributed to the verifying party.
+### CLI and node images
 
-The build process of Constellation images also creates the ground truth runtime measurements. The builds of Constellation images are reproducible and the measurements of an image can be recalculated and verified by everyone.
-With every release, Edgeless Systems publishes signed runtime measurements.
+It all starts with the CLI executable. The CLI is signed by Edgeless Systems. To ensure non-repudiability for CLI releases, Edgeless Systems publishes corresponding signatures to the public ledger of the [sigstore project](https://www.sigstore.dev/). There's a [step-by-step guide](../workflows/verify-cli.md) on how to verify CLI signatures based on sigstore.
 
-The CLI executable is also signed by Edgeless Systems.
-You can [verify its signature](../workflows/verify-cli.md).
+The CLI contains the latest runtime measurements of the Constellation node image for all supported cloud platforms. In case a different version of the node image is to be used, the corresponding runtime measurements can be fetched using the CLI's [fetch-measurements command](reference/cli#constellation-config-fetch-measurements). This command downloads the runtime measurements and the corresponding signature from cdn.confidential.cloud. See for example the following files corresponding to node image v2.16.3:
+* [Measurements](https://cdn.confidential.cloud/constellation/v2/ref/-/stream/stable/v2.16.3/image/measurements.json)
+* [Signature](https://cdn.confidential.cloud/constellation/v2/ref/-/stream/stable/v2.16.3/image/measurements.json.sig)
 
-The CLI contains the public key required to verify signed runtime measurements from Edgeless Systems.
-When a cluster is [created](../workflows/create.md) or [upgraded](../workflows/upgrade.md), the CLI automatically verifies the measurements for the selected image.
+The CLI contains the long-term public key of Edgeless Systems to verify the signature of downloaded runtime measurements.
 
-Thus, there's a chain of trust based on cryptographic signatures, which goes from CLI to runtime measurements to images. This is illustrated in the following diagram.
+### Cluster creation
+
+When a cluster is [created](../workflows/create.md), the CLI automatically verifies the runtime measurements of the *first node* (also known as "bootstrapper") using remote attestation. Based on this, the CLI and the first node set up a temporary TLS connection. This [aTLS](#attested-tls-atls) connection is used for three things:
```

Comment:
```suggestion
When a cluster is [created](../workflows/create.md), the CLI automatically verifies the runtime measurements of the *first node* (also known as "bootstrapper") using remote attestation. Based on this, the CLI and the first node set up a temporary TLS connection. This [aTLS](#attested-tls-atls) connection is used for three things:
```
The bootstrapper is an explicit program inside the VM/node. Since it is part of every VM and also used on every VM, I'd drop/rephrase the `(also known as "bootstrapper)"`